api/composer.json

```diff
@@ -70,8 +70,35 @@
         ],
         "post-update-cmd": [
             "@auto-scripts"
+        ],
+        "symfony": [
+            "php bin/console"
+        ],
+        "load-fixtures": [
+            "@symfony hautelook:fixtures:load --no-interaction --no-bundles"
+        ],
+        "prepare-test-env": [
+            "@symfony doctrine:schema:drop --env=test --force",
+            "@symfony doctrine:schema:create --env=test",
+            "@symfony messenger:setup-transports --env=test",
+            "@symfony cache:warmup --env=test"
+        ],
+        "phpunit": [
+            "bin/phpunit"
+        ],
+        "tests": [
```

Comment:
Could you adapt the [ci configuration](.github/workflows/test.yml) to use these composer scripts please?